pref-adjust.lua

```diff
@@ -1,80 +1,157 @@
--- Adjust all preferences of all dwarves in play
+-- Adjust all preferences of one or all dwarves in play
 -- by vjek
 --[====[
 
 pref-adjust
 ===========
-A two-stage script:  ``pref-adjust clear`` removes preferences from all dwarves,
-and ``pref-adjust`` inserts an 'ideal' set which is easy to satisfy::
+``pref-adjust all`` removes/changes preferences from all dwarves, and
+``pref-adjust one`` which works for a single currently selected dwarf.
+For either, the script inserts an 'ideal' set which is easy to satisfy:
 
-    Feb Idashzefon likes wild strawberries for their vivid red color,
-    fisher berries for their round shape, prickle berries for their
-    precise thorns, plump helmets for their rounded tops, prepared meals,
-    plants, drinks, doors, thrones, tables and beds. When possible, she
-    prefers to consume wild strawberries, fisher berries, prickle
-    berries, plump helmets, strawberry wine, fisher berry wine, prickle
-    berry wine, and dwarven wine.
+    ... likes iron, steel, weapons, armor, shields/bucklers and plump helmets
+    for their rounded tops.  When possible, she prefers to consume dwarven
+    wine, plump helmets, and prepared meals (quarry bush). She absolutely
+    detests trolls, buzzards, vultures and crundles.
+
+Additionally, ``pref-adjust goth`` will insert a less than ideal set, which
+is quite challenging, for a single dwarf:
+
+    ... likes dwarf skin, corpses, body parts, remains, coffins, the color
+    black, crosses, glumprongs for their living shadows and snow demons for
+    their horrifying features.  When possible, she prefers to consume sewer
+    brew, gutter cruor and bloated tubers.  She absolutely detests elves,
+    humans and dwarves.
+
+To see what values can be used with each type of preference, use
+``pref-adjust list``.  Optionally, a single dwarf or all dwarves can have
+their preferences cleared manually with the use of ``pref-adjust clear``
+and ``pref-adjust clear_all``, respectively. Existing preferences are
+automatically cleared, normally.
 
 ]====]
 
+utils = require 'utils'
+pss_counter = pss_counter or 31415926
```

Comment:
This is one potential issue I noticed - the seed would reset every time the script runs, but it's preserved across runs, so using `pss_counter or [initial value]` will only set it to the initial value if it wasn't defined in an earlier run of the script.